build.sbt

```diff
@@ -160,7 +160,16 @@ lazy val loggingDer = projectMatrix
     name := "tofu-logging-derivation"
   )
   .jvmPlatform(scala2And3Versions)
-  .dependsOn(loggingStr)
+  .dependsOn(loggingStr, loggingDerivationAnnotations)
+
+lazy val loggingDerivationAnnotations = projectMatrix
```

Comment:
You also need to add this porjetc to aggregated `logging` module here - https://github.com/tofu-tf/tofu/blob/f8d62e574b0a9b6f807525dd604ad4fbfeaeaba0/build.sbt#L262 as `(loggingDerivationAnnotations, true, true)`

Otherwise it won't be published